buildout.cfg

```diff
@@ -1,10 +1,24 @@
 [buildout]
-extends = http://download.zope.org/Zope2/index/2.13.19/versions.cfg
-parts = test
+extensions =
+    mr.developer
+extends = 
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/sources.cfg
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/versions-prod.cfg
 develop = .
+parts = 
+    test
+    tox
+auto-checkout =
+    Zope2
```

Comment:
@dataflake If 4.0a4 is not sufficient setup.py should require a version newer than it.
(Sorry mixed up the version numbers between Zope and AccessControl.)